pyproject.toml

```diff
@@ -70,7 +71,7 @@ disable = """C0330, C0301,
 duplicate-code, attribute-defined-outside-init, missing-module-docstring, missing-class-docstring,
 no-member, invalid-name, import-outside-toplevel, unused-argument, too-many-ancestors, missing-function-docstring,
 too-few-public-methods, no-self-use, too-many-arguments, cyclic-import, inconsistent-return-statements,
-useless-object-inheritance
+useless-object-inheritance, logging-fstring-interpolation
```

Comment:
Hmm I agree that we don't really benefit from deferred string formatting, which is the reason pylint wants us to avoid fstrings in logging, right?